src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -526,13 +526,15 @@ public void cannotRenewWhenLoanPolicyDoesNotExist() {
     loansFixture.checkOutByBarcode(smallAngryPlanet, jessica,
       new DateTime(2018, 4, 21, 11, 21, 43, DateTimeZone.UTC));
 
+    loanPoliciesFixture.create(unknownLoanPolicyId);
     useFallbackPolicies(
       unknownLoanPolicyId,
       requestPoliciesFixture.allowAllRequestPolicy().getId(),
       noticePoliciesFixture.activeNotice().getId(),
       overdueFinePoliciesFixture.facultyStandard().getId(),
       lostItemFeePoliciesFixture.facultyStandard().getId()
     );
+    loanPoliciesFixture.cleanUp();
```

Comment:
Please can this only delete the relevant loan policy, rather than all of them. I think that better describes this scenario.

Reply:
updated